src/main/java/games/strategy/engine/lobby/server/LobbyServer.java

```diff
@@ -68,6 +66,16 @@ public static void main(final String[] args) {
     }
   }
 
+  private static void initializeDatabase() throws SQLException {
+    if (LobbyContext.lobbyPropertyReader().isMaintenanceMode()) {
+      logger.info("Maintenance mode enabled; database may not be available");
+    } else {
+      logger.info("Starting database");
+      // TODO: this open/close may not be necessary, 'initialize the database' may be just legacy left over from derby
+      Database.getPostgresConnection().close();
```

Comment:
The question I'm asking myself here is if this additional message is actually needed, because if it isn't the message appears redundant to me.
I'm pretty sure this call to the postgres connection is not needed, but it could potentially be used as some sort of "fail early" check to ensure the db is up, although I think there must be a better way to check, because just because we're able to connect to the db doesn't mean we have all the required permissions, tables, schemas etc.
So it could still fail after this initial check passed, giving a false safety.

Reply:
Also since the properties file can be changed on-the-fly for this property, it's unlikely this mode will be turned on on a server start, and will be noticed immediately otherwhise.

Reply:
I agree with your analysis that `initializeDatabase()` doesn't really offer much benefit even as a fail-fast check.  (Although, I must admit, I regularly hit this failure when testing locally because I always forget to spin up my Postgres Docker container. :smile:)

I'm ok with removing the entire method.  Do you concur?

Reply:
No objections ^^